### PR TITLE
Initial QTMA

### LINKED ISSUE #106: Add initial QN TMA

### PR DESCRIPTION
**Pull request by motionaviation**

### CODE REVIEW

File: docs/terminal/qtma.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+|---------------------|--------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| Tower -> Approach   | Once the aircraft becomes airborne off the runway                        | Tower shall instruct all aircraft to contact QTMA airborne                                                       |
+| Approach -> Control | Once the aircraft are approaching the lateral or vertical bounds of QTMA | Aircraft may also be handed off when clear of all conflicts, provided the next sector accepts the early handover.|
+| Control -> Approach | Aircraft is approaching the vertical and lateral bounds of the TMA       | Aircraft may be handed off early, provided that QTMA is happy to accept.                                         |
+| Approach -> Tower   | Once established on approach, or within QN TWR's CTR/C.                  | QTMA may hold onto aircraft till `A050` with reference to [Arrivals](#retaining-aircraft-on-the-approach).                                 |
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
| Approach -> Tower   | Once established on approach, or within QN TWR's CTR/C.                  | QTMA may hold onto aircraft till `A050` with reference to [Arrivals](#retaining-aircraft-on-the-approach).       |

Comment (rb-vatnz) [reply docs/terminal/qtma.md]: (Last `|` to be lined up - not 100% sure my suggestion does that or not)

File: docs/terminal/qtma.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Departures
+
+Departures shall be assigned in accordance with the [SID Assignment guide](../aerodromes/Class-C/nzqn.md#sid-assignment). 
+
+QTMA may climb aircraft up to a maximum of `FL250`, with further available when handed to STH.
+
+### Track Shortening on SIDs
+
+Typically aircraft are given track shortening instructions as they climb through `A080` to track direct to a waypoint upon passing `A100`. Typically `TIMJO` and `SAVLA` for domestic flights and boundary fixes for Oceanic bound aircraft. This shall be coordinated with STH.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
Typically aircraft are given track shortening instructions as they climb through `A080` to track direct to a waypoint upon passing `A100`. This waypoint is typically `TIMJO` and `SAVLA` for domestic flights, and boundary fixes for Oceanic bound aircraft. This shall be coordinated with STH. 
```

File: docs/terminal/qtma.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Track Shortening on SIDs
+
+Typically aircraft are given track shortening instructions as they climb through `A080` to track direct to a waypoint upon passing `A100`. Typically `TIMJO` and `SAVLA` for domestic flights and boundary fixes for Oceanic bound aircraft. This shall be coordinated with STH. 
+
+## Arrivals 
+
+STH will hand aircraft to QTMA no lower than `FL180`, unless coordinated.
+
+Aircraft shall be issued either the `RNP Y (AR)` or `RNP Z (AR)` approaches. If a non-standard approach is requested, such as the `RNP F/G/H` or `VOR DME`, coordination shall occur. This requires agreement from the QTWR and QTMA.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
Aircraft shall be issued either the `RNP Y (AR)` or `RNP Z (AR)` approaches. If a non-standard approach is requested, such as the `RNP F/G/H` or `VOR DME`, coordination shall occur with agreement required between QTWR and QTMA. 
```

File: docs/terminal/qtma.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+Typically aircraft are given track shortening instructions as they climb through `A080` to track direct to a waypoint upon passing `A100`. Typically `TIMJO` and `SAVLA` for domestic flights and boundary fixes for Oceanic bound aircraft. This shall be coordinated with STH. 
+
+## Arrivals 
+
+STH will hand aircraft to QTMA no lower than `FL180`, unless coordinated.
+
+Aircraft shall be issued either the `RNP Y (AR)` or `RNP Z (AR)` approaches. If a non-standard approach is requested, such as the `RNP F/G/H` or `VOR DME`, coordination shall occur. This requires agreement from the QTWR and QTMA. 
+
+Once in QTMA airspace arrivals may be descend to `FL150` and then `A100` to commence the approach. QTMA may clear aircraft straight down to `A100` where traffic permits.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
Once in QTMA airspace, arrivals may be descended to `FL150` and then `A100` to commence the approach. QTMA may clear aircraft straight down to `A100` where traffic permits.
```

File: docs/terminal/qtma.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+!!! Note
+    Due to high mountainous terrain controllers shall avoid track shorten aircraft on the STAR. But may do so where needed for seperation with other traffic.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
    Due to high mountainous terrain, controllers shall avoid track shortening aircraft on the STAR but may do so where needed for separation with other traffic. 
```

File: docs/terminal/qtma.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+Aircraft shall be issued either the `RNP Y (AR)` or `RNP Z (AR)` approaches. If a non-standard approach is requested, such as the `RNP F/G/H` or `VOR DME`, coordination shall occur. This requires agreement from the QTWR and QTMA. 
+
+Once in QTMA airspace arrivals may be descend to `FL150` and then `A100` to commence the approach. QTMA may clear aircraft straight down to `A100` where traffic permits.
+
+!!! Note
+    Due to high mountainous terrain controllers shall avoid track shorten aircraft on the STAR. But may do so where needed for seperation with other traffic. 
+
+### Retaining Aircraft on the Approach
+
+QTMA usually elects to retain aircraft on the RNPs until they're assured separation exists at the threshold (with respect to backtracking aircraft). Down to about `A050`. This requires coordination with QTWR.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
QTMA usually elects to retain aircraft on the RNPs down to about `A050` until they are assured separation exists at the threshold (with respect to backtracking aircraft). This requires coordination with QTWR.
```

File: docs/terminal/qtma.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+### Time Seperation on the Approach
+
+When RWY 05 is in use aircraft on the `RNP Y/Z` shall be seperated by a minimum 7 minutes at `MOPGA`, the same applies to aircraft on the `RNP F` at `LETPO`.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
When RWY 05 is in use, aircraft on the `RNP Y/Z` shall be separated by a minimum of 7 minutes at `MOPGA`, or `LETPO` for aircraft on the `RNP F`.
```

File: docs/terminal/qtma.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+When RWY 23 is in use aircraft on the `RNP Y/Z` shall be seperated by a minimum of 5 minutes at `ATVUP`, the same applies to aircraft on the `RNP G/H` at `GIVAK`.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
When RWY 23 is in use, aircraft on the `RNP Y/Z` shall be separated by a minimum of 5 minutes at `ATVUP`, or `GIVAK` for aircraft on the `RNP G/H`.
```

File: docs/terminal/qtma.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+Once in QTMA airspace arrivals may be descend to `FL150` and then `A100` to commence the approach. QTMA may clear aircraft straight down to `A100` where traffic permits.
+
+!!! Note
+    Due to high mountainous terrain controllers shall avoid track shorten aircraft on the STAR. But may do so where needed for seperation with other traffic. 
+
+### Retaining Aircraft on the Approach
+
+QTMA usually elects to retain aircraft on the RNPs until they're assured separation exists at the threshold (with respect to backtracking aircraft). Down to about `A050`. This requires coordination with QTWR.
+
+### Time Seperation on the Approach
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
### Time Separation on the Approach
```

File: docs/terminal/qtma.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+When RWY 23 is in use aircraft on the `RNP Y/Z` shall be seperated by a minimum of 5 minutes at `ATVUP`, the same applies to aircraft on the `RNP G/H` at `GIVAK`.
+
+
+### Missed Approaches
+
+Aircraft that go missed that do **not** join the visual circuit will be instructed to contact QTMA passing `A040` and shall be tracked via the published missed approach to `SUNGU` to hold. Should the aircraft want to attempt another approach they shall be issued a `SUNGU` STAR for the respective runway. 
+
+!!! Note
+    Aircraft unable to reach `A092` by `SUNGU` shall be tracked further south via the missed approach track to `NV VOR` until terrain seperation can be achieved.
```
Comment (rb-vatnz) [inline docs/terminal/qtma.md]:
```suggestion
    Aircraft unable to reach `A092` by `SUNGU` shall be tracked further south via the missed approach track to `NV VOR` until terrain separation can be achieved. 
```

**Review by rb-vatnz — CHANGES_REQUESTED** [review]

**Review by rb-vatnz — APPROVED** [review]

**Review by mwr2410 — APPROVED** [review]

**Comment by mwr2410** [comment]
Ready to merge - 2412

**Pull request merged**